### PR TITLE
Plumb through compressor enable/disable

### PR DESCRIPTION
**Pull request by aschokking**

### CODE REVIEW

**Review by JohnGilb — APPROVED** [review]

File: src/main/java/xbot/common/controls/actuators/wpi_adapters/CompressorWPIAdapter.java [hunk 1]
```diff
@@ -13,4 +13,19 @@
     public CompressorWPIAdapter() {
         this.compressor = new Compressor(PneumaticsModuleType.CTREPCM);
```
Comment (stephenjust) [inline src/main/java/xbot/common/controls/actuators/wpi_adapters/CompressorWPIAdapter.java:14]: Maybe expose a property for when enabled?

Comment (aschokking) [reply src/main/java/xbot/common/controls/actuators/wpi_adapters/CompressorWPIAdapter.java:14]: there's a `isEnabled()` method, but maybe I'm missing something?

Comment (aschokking) [reply src/main/java/xbot/common/controls/actuators/wpi_adapters/CompressorWPIAdapter.java:14]: Oh did you mean a BooleanProperty?

Comment (stephenjust) [reply src/main/java/xbot/common/controls/actuators/wpi_adapters/CompressorWPIAdapter.java:14]: Yeah, add a periodic() that reports the state of isEnabled() (unless something in the base WPI impl already does it)

File: src/main/java/xbot/common/subsystems/compressor/CompressorSubsystem.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.PropertyFactory;
+
+@Singleton
+public class CompressorSubsystem extends BaseSubsystem {
+    final XCompressor compressor;
+    final BooleanProperty isEnabledProperty;
+    
+    @Inject
+    public CompressorSubsystem(CommonLibFactory clf, PropertyFactory pf) {
```
Comment (stephenjust) [inline src/main/java/xbot/common/subsystems/compressor/CompressorSubsystem.java:20]: I think you need to this.register()

**Review by stephenjust — CHANGES_REQUESTED** [review]

**Review by stephenjust — APPROVED** [review]

**Pull request merged**